sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -194,7 +196,7 @@ async def serialize_property(value: 'Input[Any]',
         value = _types.input_type_to_dict(value)
         transform_keys = False
 
-    if isinstance(value, Mapping):  # pylint: disable=bad-option-value,isinstance-second-argument-not-valid-type
+    if isinstance(value, abc.Mapping):
```

Comment:
While the previous code worked, we should be checking against the abstract base class from `collections.abc` rather than the type from `typing`.